src/header.wppl

```diff
@@ -469,11 +469,49 @@ var OptimizeThenSample = function(wpplFn, options) {
   return SampleGuide(wpplFn, opts);
 };
 
-var Infer = function(options, maybeFn) {
-  if (!util.isObject(options)) {
-    util.fatal('Infer: expected first argument to be an options object.');
+/*
+ * DefaultInfer() called when no options are specified
+ * var enumTol: upper bound for enumeration tree size, enumeration ends when above threshold.
+ * var minSampleRate: lower bound for #sample/sec, rejection ends when below threshold.
```

Comment:
Use `samples` instead of `sampleNum` below, to make it analogous to the name we use for the corresponding MCMC/Rejection option?